server/server.go

```diff
@@ -415,20 +431,62 @@ func (s *Server) v0QueryPath(w http.ResponseWriter, r *http.Request, path ast.Re
 	writer.JSON(w, 200, qrs[0].Result, false)
 }
 
-func (s *Server) v1DataGet(w http.ResponseWriter, r *http.Request) {
+func (s *Server) returnDiagnostics(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: this function is interleaved with a bunch of the server's handlers...I'd probably move it to the bottom of the file with the rest of the helper functions.